projects/laji/src/app/+theme/theme.component.html

```diff
@@ -48,6 +48,10 @@ <h5>{{ 'navigation.lolife' | translate }}</h5>
       <h5>{{ 'navigation.syke-butterfly' | translate }}</h5>
       <p>{{ 'theme.syke-butterfly.intro' | translate }}</p>
     </a>
+    <a [routerLink]="['/project/' + Global.oldThemeRouting['kimalaiset']] | localize" tabindex="-1" class="navbox">
```

Comment:
Could you remove the `tabindex` attribute? They will be removed from all other theme items in https://github.com/luomus/laji/pull/114